packages/react-components/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@typespec/react-components",
+  "version": "0.57.0",
+  "author": "Microsoft Corporation",
+  "description": "TypeSpec library for emitting an html view of the program.",
+  "homepage": "https://typespec.io",
+  "readme": "https://github.com/microsoft/typespec/blob/main/README.md",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/microsoft/typespec.git"
+  },
+  "bugs": {
+    "url": "https://github.com/microsoft/typespec/issues"
+  },
+  "keywords": [
+    "typespec"
+  ],
+  "type": "module",
+  "main": "dist/src/index.js",
```

Comment:
`dist/src/` isn't being generated since the test files aren't included in the tsconfig.